HW1/P3/P3.py

```diff
@@ -0,0 +1,20 @@
+from pyspark import SparkContext, SparkConf
+
+#Setup
+conf = SparkConf().setAppName("anagrams").setMaster("local[*]")
+sc = SparkContext(conf=conf)
+
+#For each word, create a key out of its ordered sequence of letters
+words = sc.textFile('EOWL_words.txt')
+sortedLetters_words = words.map(lambda w: (tuple(sorted(w)), [w]))
```

Comment:
Nice implementation of tuple. There is a performance gain by using tuple instead of list.